lib/vdsm/storage/lvm.py

```diff
@@ -746,7 +746,7 @@ def _reloadvgs(self, vgName=None):
 
         return updatedVGs
 
-    def _updatelvs_locked(self, lvs_output, vg_name, lv_names):
+    def _updatelvs_locked(self, lvs_output, vg_name):
```

Comment:
This should be the second patch after removing renameLV. Do you want to split this work to another PR to make it easier and quicker?

Adding more changes to this PR will only make it slower to review. Lets finish first with removal of dead code and options that we don't need. When we have simpler code it will be easier to refactor it safely.

Reply:
Yeah, good idea, I'll split this PR.